token-client/src/main/java/com/sap/cloud/security/xsuaa/tokenflows/JwtBearerTokenFlow.java

```diff
@@ -83,14 +116,16 @@ public JwtBearerTokenFlow subdomain(String subdomain) {
     }
 
     /**
-     * Adds additional authorization attributes to the request.
+     * Adds additional authorization attributes to the request. <br>
+     * Clients can use this to request additional attributes in the
+     * 'az_attr' claim of the returned token.
      *
-     * @param optionalParameters
-     *            - the optional parameters.
+     * @param additionalAuthorizationAttributes
+     *            - the additional attributes.
      * @return this builder.
      */
-    public JwtBearerTokenFlow optionalParameters(Map<String, String> optionalParameters) {
-        this.optionalParameters = optionalParameters;
+    public JwtBearerTokenFlow attributes(Map<String, String> additionalAuthorizationAttributes) {
```

Comment:
@liga-oz Do you plan to also apply the same change on `main-dev`? Because otherwise we would have a breaking change here between v2 and v3, right?